AutomatedTesting/Gem/PythonTests/scripting/EditMenu_Default_UndoRedo.py

```diff
@@ -49,58 +35,39 @@ class EditMenu_Default_UndoRedo:
 
     :return: None
     """
-    def __init__(self):
-        self.editor_main_window = None
-        self.sc_editor = None
-        self.sc_editor_main_window = None
-        self.variable_manager = None
-
-    def get_graph_variables_row_count(self, graph_variables):
-        return graph_variables.model().rowCount(QtCore.QModelIndex())
-
-
-    @pyside_utils.wrap_async
-    async def run_test(self):
 
-        # Preconditions
-        general.idle_enable(True)
-        scripting_tools.initialize_editor_object(self)
+    # Preconditions
+    import azlmbr.legacy.general as general
+    import editor_python_test_tools.script_canvas_tools_qt as sc_tools_qt
 
-        # 1) Open Script Canvas window and initialize the qt SC objects
-        general.open_pane(SCRIPT_CANVAS_UI)
-        helper.wait_for_condition(lambda: general.is_pane_visible(SCRIPT_CANVAS_UI), WAIT_TIME_3)
-        scripting_tools.initialize_sc_editor_objects(self)
+    general.idle_enable(True)
 
-        # 2) Open Variable Manager if not opened already
-        scripting_tools.initialize_variable_manager_object(self)
+    # 1) Open Script Canvas window
+    qtpy_o3de_editor = sc_tools_qt.qtpy_o3de_editor
+    qtpy_o3de_editor.open_script_canvas()
 
-        # 3) Create Graph
-        scripting_tools.create_new_sc_graph(self.sc_editor_main_window)
+    # 2) Create Graph
+    sc_editor = qtpy_o3de_editor.sc_editor
+    sc_editor.create_new_script_canvas_graph()
 
-        # 4) Create and verify the new variable exists in variable manager
-        scripting_tools.create_new_variable(self, VARIABLE_TYPES[0])
-        graph_variables = self.variable_manager.findChild(QtWidgets.QTableView, GRAPH_VARIABLES_QT)
-        row_count = 1
-        Report.result(Tests.variable_created, helper.wait_for_condition(
-            lambda: self.get_graph_variables_row_count(graph_variables) == row_count, WAIT_TIME_3))
+    # 3) Create and verify the new variable exists in variable manager
+    qtpy_variable_manager = sc_editor.variable_manager
+    variable_types = qtpy_variable_manager.get_basic_variable_types()
```

Comment:
Let's have this be part of the initializer of the variable manager that we can then reference through `variable_manager.types.Boolean` without having to initialize in a test file.

Reply:
a simple namespace variable now lives on the variable manager object. it gets initialized on variable manager construction